lib/stale.js

```diff
@@ -19,7 +19,8 @@ module.exports = class Stale {
   }
 
   async markAndSweep (type) {
-    const {only} = this.config
+    const only = this.getConfigValue(type, 'only')
```

Comment:
Hmm, `getConfigValue` is meant for pulling values for specific types and falling back to the default. I don't think it should be needed here.

Reply:
Done. I removed the last two commits, but kept two tests in a new commit.